features/shopcart.feature

```diff
@@ -74,7 +77,24 @@ Scenario: Delete
     And I press the "Read" button
     Then I should see the message "Read shopcart Failed"
 
-#Scenario: Update 
+Scenario: Update 
+	When I visit the "Home Page"
+	And I set the "Customer_ID" to "2"
+	And I set the "Product_ID" to "3"
+	And I set the "Quantity" to "450"
+	And I press the "Update" button
+	Then I should see the message "Success"
+	When I press the "Clear" button
```

Comment:
Can you check for the indentation or is it just on GitHub?

Reply:
what do you mean?

Reply:
scenario for list and query were commented so i uncommented them if that's what you are asking